ports/vcpkg-cmake/vcpkg.json

```
@@ -1,6 +1,6 @@
 {
   "name": "vcpkg-cmake",
-  "version-date": "2023-05-04",
-  "documentation": "https://vcpkg.io/en/docs/maintainers/ports/vcpkg-cmake.html",
+  "version-date": "2024-04-18",
+  "documentation": "https://learn.microsoft.com/vcpkg/maintainers/authoring-script-ports",
```

```suggestion
  "documentation": "https://learn.microsoft.com/vcpkg/maintainers/functions/vcpkg_cmake_install",
```
`authoring-script-ports` covers generically authoring any script port. While it would be good to have a dedicated landing page for the `vcpkg-cmake` helper all-up, I think for now we should link to the "primary" helper function's documentation.

---

IMO the primary helper function is `vcpkg_cmake_configure` which does all the heavy lifting and receives all the improvements. `vpkg_cmake_install` isn't much more than a thin wrapper for running `ninja`.

---

SGTM, I'd be happy to see a link to https://learn.microsoft.com/vcpkg/maintainers/functions/vcpkg_cmake_configure.